cli/lib.rs

```diff
@@ -393,7 +393,10 @@ async fn test_command(
   }
 
   let test_file = test_runner::render_test_file(test_modules, fail_fast);
-  let test_file_path = cwd.join(".deno.test.ts");
+  // Interpolated with pid, because multiple deno process may be run in current
+  // directory to run tests.
+  let filename = format!(".deno.{}.test.ts", std::process::id());
+  let test_file_path = cwd.join(filename);
```

Comment:
Is there a good reason to place this in the cwd? Why not a temp file?

Reply:
Yeah, I think it was to avoid recompilation for repeat `deno test` in the same conditions but if the name is different each time that doesn't apply anymore.

Reply:
@nayeemrmn good point, now I forgot about it :/ I hit issue of name collision when I added two integration tests for runner - both of them create file almost simultaneously and one of the test is failing.

Reply:
Removed writing to fs completely, used the same logic as for `deno eval` and `deno` (repl). Let's see how this works out